tests/test_connectionstr.py

```diff
@@ -13,18 +13,18 @@
 
 def test_empty_connectionstring():
     with pytest.raises(KeyError):
-        connection_str_dict = Utils.parse_device_connection_str(empty_string)
+        connection_str_dict = Utils.parse_connection_strs(empty_string)
         assert not connection_str_dict
 
 
 def test_valid_connectionstring():
```

Comment:
Add iot hub connection string to test case?

Reply:
it's in test_utils